qutebrowser/browser/pdfjs.py

```diff
@@ -49,24 +50,25 @@ def generate_pdfjs_page(filename, url):
         filename: The filename of the PDF to open.
         url: The URL being opened.
     """
-    if not is_available():
+    pdfjs_name = _get_pdfjs_basename()
+    if pdfjs_name is None or not is_available():
         pdfjs_dir = os.path.join(standarddir.data(), 'pdfjs')
         return jinja.render('no_pdfjs.html',
                             url=url.toDisplayString(),
                             title="PDF.js not found",
                             pdfjs_dir=pdfjs_dir)
-    html = get_pdfjs_res('web/viewer.html').decode('utf-8')
+    viewhtml = get_pdfjs_res('web/viewer.html').decode('utf-8')
 
     script = _generate_pdfjs_script(filename)
-    html = html.replace('</body>',
+    viewhtml = viewhtml.replace('</body>',
                         '</body><script>{}</script>'.format(script))
     # WORKAROUND for the fact that PDF.js tries to use the Fetch API even with
     # qute:// URLs.
-    pdfjs_script = '<script src="../build/pdf.js"></script>'
-    html = html.replace(pdfjs_script,
+    pdfjs_script = f'<script src="../build/{html.escape(pdfjs_name)}"></script>'
```

Comment:
I don't think this works anymore because when using modules the source is now:

    <script src="../build/pdf.mjs" type="module"></script>
    
One way to fix it is to add a `{attrs}` variable in the f-string that is set to ` "type="module"` if the name ends with `.mjs`, otherwise it's empty.
An even easier way to fix it is to just put the workaround after `head` instead of after the script tag.

But I don't think this workaround is even active on the current pdfjs version anyhow. The code from here doesn't seem to be a thing anymore: https://github.com/qutebrowser/qutebrowser/issues/4235#issuecomment-423520775